angularSTEP/src/app/discover-page/discover-page.component.html

```diff
@@ -71,12 +71,12 @@
     <!-- displays the user search bar -->
     <div *ngIf='this.searchOption[0] == "Users"' class="content">
       <ais-instantsearch [config]="configUsers">
-        <app-search-box (noQuery)='displayResults($event)' contentType='Search for users' [delay]='500'></app-search-box>
+        <app-search-box (noQuery)='displayResults($event)' [searchField]='query' contentType='Search for users' [delay]='500'></app-search-box>
           <ais-hits *ngIf='showResults'>
             <ng-template let-hits="hits">
                 <div class="search-content-container">
                   <app-profile-card *ngFor="let user of hits" (click)='goToUser(user.username)'
-                    [user]='{displayName: user.displayName, username: user.username, picUrl: user.picUrl}'>
+                    [user]='user'>
```

Comment:
What a nice simplification! Just curious, how did this pattern of recreating another object with the same fields get started?

Reply:
That pattern was actually from before we created the centralized User and Recipe class. I only realized now though that it still had yet to be changed.